src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -653,12 +653,12 @@ class JsonPrinter : public Printer {
         BSLS_KEYWORD_OVERRIDE
     {
         closeBraceIfOpen();
-        d_ostream << "  \"TotalMessagesNumber\": \"" << totalMessagesCount
-                  << "\",\n  \"PartiallyConfirmedMessagesNumber\": \""
+        d_ostream << "  \"TotalMessagesNumber\": " << totalMessagesCount
+                  << ",\n  \"PartiallyConfirmedMessagesNumber\": "
                   << partiallyConfirmedCount
-                  << "\",\n  \"ConfirmedMessagesNumber\": \"" << confirmedCount
-                  << "\",\n  \"OutstandingMessagesNumber\": \""
-                  << outstandingCount << "\"";
+                  << ",\n  \"ConfirmedMessagesNumber\": " << confirmedCount
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.